data/hai/hai reveal 5 philosophers.txt

```diff
@@ -732,37 +767,97 @@ mission "Hai Reveal [B11] Moving Together"
-			`	Remington takes over again. "However, now time is of the essence. This has gone on long enough that the popular opinion is that the Hai must obviously exist, even though there are no explicit voices saying as such. It has in fact almost become a joke; the perception being that, 'The Navy are blockading the Hai to hide humanity from them, because the blockade isn't doing a good job of being inconspicuous.' There have been memes of the Navy hiding their ships behind far too narrow light poles while they distract a squirrel with their headlights. Now it's a matter of being able to present good news. Place a successful spin on things."`
-			`	"Our problem," says Khaine. "Is Elenchus.`
-			`	"Reaching Elenchus would seem to be a problem. Humanity doesn't have many jump drives, and the owners of them can't or won't risk all of them to try and take a base which has surface defenses. The Hai inform us that they do not have jump drives at all, though the Unfettered do. Additionally, the base on Elenchus appears to be a hardened target. It will need specialized and suitably equipped shock troops to be sure of taking it. In our assessment so far we have determined that the Syndicate have suitable troopers they can mobilize quickly, but no way of getting them there, and our suitable shock troops are largely pre-occupied either being needed here on <origin>, or are at least a week away from being available. We need a resolution sooner than that."`
+				goto briefing
+			label secret
+			action
+				set "Hai Reveal [B11] Moving Together: label secret"
+			`	Remington explains. "This operation has gone on long enough that the popular opinion is that the Hai must obviously exist, even though there are no explicit voices saying as such. It has in fact almost become a joke; the perception being that, 'The Navy are blockading the Hai to hide humanity from them, because the blockade isn't doing a good job of being inconspicuous.' There have been memes of the Navy hiding their ships behind far too narrow light poles while they distract a squirrel with their headlights. Now it's a matter of being able to present good news. Place a successful spin on things."`
+			`	"Our problem," says Khaine. "Is Elenchus."`
+				goto briefing
+			label elenchus
+			action
+				set "Hai Reveal [B11] Moving Together: label elenchus"
+			`	Admiral Kaine explains. "Reaching Elenchus would seem to be a problem. The Unfettered are willing to loan us a few jump drives. Additionally, the base on Elenchus appears to be a hardened target. It will need specialized and suitably equipped shock troops to be sure of taking it. In our assessment so far we have determined that the Syndicate have suitable troopers they can mobilize quickly, but no way of getting them there, and our suitable shock troops are largely pre-occupied either being needed here on <origin>, or are at least a week away from being available. We need a resolution sooner than that."`
```

Comment:
```suggestion
			`	Marshal Kaine explains, "Reaching Elenchus would seem to be a problem, because it is behind systems we cannot reach with our hyperdrives. However, the Unfettered are willing to loan us a few jump drives. Additionally, the base on Elenchus appears to be a hardened target. It will need specialized and suitably equipped shock troops to be sure of taking it. In our assessment so far we have determined that the Syndicate have suitable troopers they can mobilize quickly, but no way of getting them there, and our suitable shock troops are largely pre-occupied; either being needed here on <origin>, or are at least a week away from being available. We need a resolution sooner than that."`
```